apps/test/unit/templates/certificates/CertificateShareTest.js

```diff
@@ -31,28 +32,30 @@ describe('CertificateShare', () => {
   });
 
   it('renders announcement image url relative to pegasus', () => {
-    const wrapper = shallow(<CertificateShare {...defaultProps} />);
-
-    const printLink = wrapper.find('a');
-    expect(printLink.prop('href')).to.equal('/certificate-print');
-    const image = printLink.find('img');
-    expect(image.prop('src')).to.equal('/certificate-image');
-
-    const block = wrapper.find('TwoColumnActionBlock');
-    expect(block.prop('imageUrl')).to.equal('//code.org/announcement-image');
+    render(<CertificateShare {...defaultProps} />);
+
+    const printLink = screen.getByRole('link', {name: 'certificate alt text'});
```

Comment:
Nit: for the uses of `certificate alt text` in the tests, would using `defaultProps.imageAlt` help in case we change it later or do you think writing out the string is more readable? I'm good either way just wanted to ask!